### PR TITLE
Added the option to show examples for arrays #1676

### PR DESCRIPTION
**Pull request by sturlath**
Just a start to get something working that can be descussed. Will add more tests to cover all scenarios and refine the code.

### CODE REVIEW

File: src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/OpenApiAnyFactory.cs [hunk 1]
```diff
@@ -53,5 +57,66 @@ private static bool TryCast<T>(object value, out T typedValue)
...
+                {
+                    array.Add(item: new OpenApiLong(longResult));
+                }
+                else if (double.TryParse(item, out double doubleResult))
+                {
+                    array.Add(item: new OpenApiDouble(doubleResult));
+                }
+                else if (DateTime.TryParse(item, out DateTime dateTimeResult))
+                {
+                    //TODO: What is the difference between OpenApiDate and OpenApiDateTime?
```
Comment (sturlath) [inline src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/OpenApiAnyFactory.cs:105]: Is it just Date with and without time? How does that effect the examples?

**Comment by mattfrear** [comment]
Just a heads up that https://github.com/domaindrivendev/Swashbuckle.AspNetCore/pull/1678 is also open, and changes how the examples on parameters are done.

**Comment by TAGC** [comment]
Any progress on this?

**Comment by Aandree5** [comment]
Any news? Would really like this feature

**Comment by domaindrivendev** [comment]
@Aandree5 - I believe this functionality is already merged in, albeit with a different implementation, as a result of #1866, although not yet released as an official Nuget. I'm doing a pass through all of the open PRs and will be releasing a new version when that's done. In the meantime, you can pull down a preview package from myget.org (see https://myget.org/feed/domaindrivendev/package/nuget/Swashbuckle.AspNetCore) to test out the functionality.

**Pull request closed without merging**